core/chains/evm/txmgr/common.go

```diff
@@ -65,21 +63,9 @@ func batchSendTransactions(
 			return reqs, errors.Wrap(err, "failed to batch send transactions")
 		}
 
-		if err := updateBroadcastAts(db, now, ethTxIDs[i:j]); err != nil {
+		if err := (*orm).UpdateBroadcastAts(now, ethTxIDs[i:j]); err != nil {
```

Comment:
is there a prettier way to invoke methods of a pointer to an interface?

Reply:
Having a pointer to an interface at all is pretty unusual. Can the interface type be used directly instead?

Reply:
I can def use the interface directly. My initial hesitancy to do so was to prevent copy by values.

Reply:
https://stackoverflow.com/questions/44370277/type-is-pointer-to-interface-not-interface-confusion helped clarify it for me. An interface can hold a pointer to a struct 🚀